spec/models/reservation_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
- [ ] Please, add tests and fix this error, this happened when I run the test with `rspec`. 🐱 
<img width="701" alt="image" src="https://user-images.githubusercontent.com/20563252/176566831-1e6dcc47-2d6b-49fc-8cf6-e2b55c6dacf2.png">